src/support_sphere/lib/main.dart

```diff
@@ -25,8 +31,20 @@ class MyApp extends StatelessWidget {
         Brightness.light,
       ),
 
-      // Landing page
-      home: const Landing(),
+      // Routing configuration
+      initialRoute: '/',
+      onGenerateRoute: AppRouter.onGenerateRoute,
+      onUnknownRoute: (settings) {
+        // Handle unknown routes
+        // essentially a 404 page
+        return MaterialPageRoute(
+          builder: (_) => Scaffold(
+            body: Center(
+              child: Text('404 NOT FOUND: No route defined for ${settings.name}'),
```

Comment:
Should this be a constant/are we going to reuse this? Is exposing that implementation detail of an undefined route to a user the right messaging for this case?

Open questions, definitely not blocking :)

Reply:
It's a place holder for now.. probably will just think about this more later 😄